marketplace/home/templates/index.html

```diff
@@ -1,9 +1,31 @@
+<!DOCTYPE html>
 <html>
 <head>
-    <title>Home</title>
+    <title>Marketplace | Home</title>
+    <!-- Include Bootstrap CSS -->
+    {% load static %}
+    <link rel="stylesheet" href="{% static 'css/bootstrap.min.css' %}">
 </head>
 <body>
-    <h1>Home</h1>
-    <p>Home page</p>
+    <!-- Header Section -->
+    <header>
+        <nav class="navbar navbar-expand-lg navbar-light bg-light">
+            <a class="navbar-brand" href="#">
+                
+                <img width="80px" src="{% static 'img/logo.jpg' %}" alt="">
+            </a>
+        </nav>
+    </header>
+
+    <!-- Main Content Section -->
+    <div class="container">
+        <h1>Welcome to the Marketplace!</h1>
+        <p>Discover amazing products and connect with sellers.</p>
+        <a href="#" class="btn btn-primary">Browse Products</a>
+    </div>
+
+    <!-- Include Bootstrap JS -->
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"></script>
```

Comment:
Remove this.

Reply:
Resolved the issue..
removed bootstrap js file.

Reply:
Commit it please.